gems/aws-crt/lib/aws-crt/platforms.rb

```diff
@@ -25,18 +25,14 @@ def crt_bin_name(platform)
   OS_BINARIES[platform.os] || DEFAULT_BINARY
 end
 
-# @return [String] return the path to the output binary from building
-def crt_build_out_path(platform)
-  PLATFORM_BUILD_PATHS[platform.to_s] ||
-    "native/build/#{crt_bin_name(platform)}"
+# @return [String] return the directory of the CRT library for the platform
+def crt_bin_dir(platform)
+  File.expand_path("../../bin/#{platform.cpu}", File.dirname(__FILE__))
```

Comment:
nice.  We can probably remove the `PLATFORM_BUILD_PATHS` as well.

Reply:
gone